### PR TITLE
OGM-718 Make sure RESOURCE_LOCAL persistence unit do not require a transaction manager

### PR DESCRIPTION
**Pull request by hferentschik**

### CODE REVIEW

**Comment by emmanuelbernard** [comment]
Review done. Looks good overall. The rest is in local comments.

File: core/src/main/java/org/hibernate/ogm/service/impl/OgmIntegrator.java [hunk 1]
```diff
@@ -85,6 +114,17 @@ private void doIntegrate(Configuration configuration, SessionFactoryImplementor
 		attachPersistListener( serviceRegistry );
 	}
 
+	private boolean isOgmUsed(Map properties) {
+		// Integrator are discovered via the Java ServiceLoader mechanism and get executed independently of whether a
```
Comment (gunnarmorling) [inline core/src/main/java/org/hibernate/ogm/service/impl/OgmIntegrator.java]: Ah, very nice. I still meant to create a separate issue for it, but you are ahead of me :)

Comment (hferentschik) [inline core/src/main/java/org/hibernate/ogm/service/impl/OgmIntegrator.java]: :-)

**Comment by hferentschik** [comment]
Great feedback. I'll make the suggested changes and push an updated pull request...

**Comment by hferentschik** [comment]
Superseded by #504

**Pull request closed without merging**